vignettes/advanced.Rmd

```diff
@@ -26,102 +29,290 @@ knitr::opts_chunk$set(
 The purpose of this vignette is to provide an overview of some of the more advanced
 features of the `rbmi` package. 
 
+# Data simulation {#sec:dataSimul}
+
+In order to demonstrate the advanced functions we will first create a simulated dataset with the `rbmi` function `simulate_data()`.
+A reader interested in other sections of the vignette must not necessarly read through this section.
+The `simulate_data()` function generates data for a two-arms clinical trial with longitudinal continuous outcome and two intercurrent events (ICEs).
+One intercurrent event (ICE1) may be thought of as a discontinuation from study treatment due to study drug or condition related (SDCR) reasons.
```

Comment:
"may be" ?  Surely that is what it is ?

Reply:
Perhaps, "The ICE1 intercurrent event represents discontinuation from ..."

Reply:
mmm, strong preference on this? 

I think sometimes one could model similarly other type of intercurrent events through ICE1. ICE1 represents an ICE that happens more or less likely depending on the current outcome value. Sometimes this could also be e.g start of symptomatic treatment? Could it be used to model death in certain settings? (even though the post-ice1 trajectory is done under a reference-based method, this usually makes sense mainly for treatment discontinuation...)

Reply:
We could also just be neutral and say that the occurrence of ICE1 may depend on the observed outcome value and that the function allows to simulate post-ICE1 outcomes under reference-based assumptions. For example, ICE1 could be a a discontinuation from study treatment due to study drug or condition related (SDCR) reasons.